src/implementations/twig/components/blockquote/blockquote.story.js

```diff
@@ -40,10 +41,28 @@ const getArgTypes = () => {
         type: 'text',
       },
     },
+    show_image: {
+      name: 'show_image',
+      type: { name: 'boolean' },
+      defaultValue: false,
```

Comment:
i see you added the show_image default value correctly in the args object, this one should be removed, then.